components/collection/CollectionGrid.vue

```diff
@@ -75,7 +75,7 @@ const route = useRoute()
 const { accountId } = useAuth()
 const { urlPrefix, client } = usePrefix()
 const preferencesStore = usePreferencesStore()
-
+const { artGenModeEnabled } = useArtGenMode()
```

Comment:
```suggestion
const { genArtModeEnabled } = useGenArtMode()
```

since kind is `genart`, wdyt?


edit: I see that maybe because of `artView` ?

Reply:
updated, I have no preference on this naming.